src/img/acperm_bit_field.edn

```diff
@@ -15,7 +15,9 @@
 (draw-box "Reserved" {:span 6})
 (draw-box "SDP"      {:span 4})
 (draw-box "C"        {:span 1})
-(draw-box "Reserved" {:span 3})
+(draw-box "CL"       {:span 1})
+(draw-box "SL"       {:span 1})
+(draw-box "EL"       {:span 1})
```

Comment:
The indexes and width of these fields need to be adjusted because the figure is not printing correctly.